examples/core/core_2d_camera_smooth_pixelperfect.c

```diff
@@ -75,15 +75,15 @@ int main(void)
         {
             worldSpaceCamera.target.x = (int)screenSpaceCamera.target.x;
             screenSpaceCamera.target.x -= worldSpaceCamera.target.x;
-            screenSpaceCamera.target.x *= virtualRatio;
+            screenSpaceCamera.target.x*=virtualRatio;

```

Comment:
Sorry... but between `*=` I use spaces...

Reply:
@NotManyIdeasDev Excuse me for being so picky... I can just merge and review those little details myself...

Reply:
xD, hope this is the last one!